Sources/VaporSecurityHeaders/Configurations/ReferrerPolicyConfiguration.swift

```diff
@@ -14,13 +14,17 @@ public struct ReferrerPolicyConfiguration: SecurityHeaderConfiguration {
         case unsafeUrl = "unsafe-url"
     }
 
-    private let option: Options
+    private let headerValue: String
```

Comment:
I'd prefer to keep the type safety here
```suggestion
    private let options: [Directive]
```